packages/oruga/src/components/datepicker/DatepickerMonth.vue

```diff
@@ -0,0 +1,467 @@
+<template>
+    <section :class="monthClasses">
+        <div :class="monthBodyClasses">
+            <div class="datepicker-months">
+                <template v-for="(date, index) in monthDates">
+                    <a
+                        :ref="`month-${date.getMonth()}`"
+                        v-if="selectableDate(date) && !disabled"
+                        :key="index"
+                        :class="cellClasses"
+                        role="button"
+                        href="#"
+                        :disabled="disabled"
+                        @click.prevent="updateSelectedDate(date)"
+                        @mouseenter="setRangeHoverEndDate(date)"
+                        @keydown.prevent="manageKeydown($event, date)"
+                        :tabindex="focused.month === date.getMonth() ? null : -1">
+                        {{ monthNames[date.getMonth()] }}
+                        <div class="events" v-if="eventsDateMatch(date)">
+                            <div
+                                class="event"
+                                :class="event.type"
+                                v-for="(event, index) in eventsDateMatch(date)"
+                                :key="index"/>
+                        </div>
+                    </a>
+                    <div
+                        v-else
+                        :key="index"
+                        :class="cellClasses(date)">
+                        {{ monthNames[date.getMonth()] }}
+                    </div>
+                </template>
+            </div>
+        </div>
+    </section>
+</template>
+
+<script>
+import { isDefined } from '../../utils/helpers'
+
+import BaseComponentMixin from '../../utils/BaseComponentMixin'
+import DatepickerMixin from './DatepickerMixin'
+
+export default {
+    name: 'ODatepickerMonth',
+    mixins: [BaseComponentMixin, DatepickerMixin],
+    props: {
+        value: {
+            type: [Date, Array]
+        },
+        monthNames: Array,
+        events: Array,
+        indicators: String,
+        minDate: Date,
+        maxDate: Date,
+        focused: Object,
+        disabled: Boolean,
+        dateCreator: Function,
+        unselectableDates: [Array, Function],
+        unselectableDaysOfWeek: Array,
+        selectableDates: [Array, Function],
+        range: Boolean,
+        multiple: Boolean,
+        monthClass: [String, Function, Array],
+        monthBodyClass: [String, Function, Array],
```

Comment:
You added new *class props but you didn't add the class implementation into .scss, why?
Also, you should add the same props to Datepicker in order to pass them from datepicker instance

Reply:
I miss this thing, I'll try to fix it

Reply:
done

Reply:
![image](https://user-images.githubusercontent.com/8029488/142771076-5c5d567c-6478-488e-9872-c29094eb2a03.png)

Probably there are other missing classes to apply

Reply:
@jtommy how can I test visually the component (like you done for creating this screenshot) ?

Reply:
fixed the missing style class.
The layout is not still correct, but the problem is that the cells are not populated with `o-dpck__month_cell` class and this is a problem related with the failure of the unit tests
![Screenshot 2021-11-23 at 10-27-34 UI components for Vue js and CSS framework agnostic](https://user-images.githubusercontent.com/4171821/143000187-0f1f4b73-cc63-44a0-b597-311325c86454.png)
.

Reply:
Why?

Reply:
There isn't any space between months labels because all the links representing the months have the class attribute empty.
If they would have the `o-dpck__month__cell` the layout would be correct.
I'll try to do some rude debug to understand what happens

Reply:
Fixed the style problem and now the tests are ok (except the two in comments)